receiver/sqlserverreceiver/metadata.yaml

```
@@ -255,6 +263,62 @@ metrics:
       input_type: string
     attributes: [database.status]
     extended_documentation: This metric is only available when the receiver is configured to directly connect to SQL Server.
+  sqlserver.query.execution_count:
+    enabled: false
+    description: Number of executions of the query
+    unit: "{executions}"
+    sum:
+      value_type: double
+    extended_documentation: This metric is only available when the receiver is configured to directly connect to SQL Server.
+  sqlserver.query.total_elapsed_time:
+    enabled: false
+    description: The total time taken by the query
+    unit: "s"
+    sum:
+      value_type: double
+    extended_documentation: This metric is only available when the receiver is configured to directly connect to SQL Server.
+  sqlserver.query.total_rows:
```

```suggestion
  sqlserver.query.returned_rows:
```

I think this will be a better name.

Related discussion: https://github.com/open-telemetry/semantic-conventions/pull/1438#discussion_r1811797666
Related semantic convention: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/database/database-metrics.md#metric-dbclientresponsereturned_rows